extensions/Worker.Extensions.Storage.Tables/src/Worker.Extensions.Storage.Tables.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables</RootNamespace>
+    <Description>Azure Table Storage extensions for .NET isolated functions</Description>
+    
+    <!--Version information-->
+    <VersionPrefix>1.0.0</VersionPrefix>
+
+    <!--Temporarily opting out of documentation. Pending documentation-->
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
```

Comment:
It looks like the attributes have been documented. Should we still be opting out here? What else are we missing?